Library/Formula/terminal-notifier.rb

```diff
@@ -27,6 +27,6 @@ def install
   end
 
   test do
-    system "#{bin}/terminal-notifier", "-message", "Hello World!"
+    system "#{bin}/terminal-notifier", "-help"
```

Comment:
I don't think `if MacOS.version < :yosemite` is a good fix. The test failed is not Yosemite specific. It is related to our CI settings. So in the future, this test can fail again in different OS. 

> is there anything I can do in terminal-notifier

It would be great that if you can investigate why this fails in the headless environment(like invoking command over ssh). Or can you just stop terminal-notifier connecting to WindowServer when only asking help message. 

If we cannot fix it, I would suggest to kill test completely. From the git log, we did this before in this formula.